RetailCoder.VBE/Inspections/QuickFixes/PassParameterByReferenceQuickFix.cs

```diff
@@ -15,11 +15,13 @@ namespace Rubberduck.Inspections.QuickFixes
     public class PassParameterByReferenceQuickFix : QuickFixBase
     {
         private Declaration _target;
+        private AssignedByValParameterQuickFixHelper _quickFixHlpr;
```

Comment:
I like the idea (very much!) of moving code out of quickfix implementations and into a common "helper" class, however it needs a more specialized name than "helper" (I'm thinking `CodePaneRewriter` or something); know however that we are planning to come up with an actual API specifically for this (ref. #2688), so I think I'd rather have efforts concentrated more toward shaping that API than refactoring things into a temporary helper that will end up superseded by that API - although, that helper can help us figure out everything this API needs to be able to do.